### PR TITLE
#feat deep search result component

### PR DESCRIPTION
**Pull request by dkurgan**

### CODE REVIEW

File: src/components/KeywordsDisplay/index.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { Tooltip } from '@material-ui/core';
 import shuffleArray from '../helpers/shuffleArray'
 
 export const KeywordsDisplay = React.memo(({ keywords }) => {
-    const keywordsToMap = shuffleArray(keywords).slice(0,6)
+    if (!keywords && keywords === undefined) return;
```
Comment (ivankozlovcodes) [inline src/components/KeywordsDisplay/index.js:7]: what `!keywords` is doing here?

File: src/components/deepSearchPage/submitForm/DeepSearchForm.js [hunk 2]
```diff
@@ -78,11 +78,16 @@ const DeepSearchForm = () => {
 
     try {
       const name = `${firstName} ${lastName}`;
-
-      const data = await fetchDeepSearch({
+      console.log({
```
Comment (ivankozlovcodes) [inline src/components/deepSearchPage/submitForm/DeepSearchForm.js:81]: console log

File: src/components/deepSearchPage/submitForm/DeepSearchForm.js [hunk 3]
```diff
@@ -78,11 +78,16 @@ const DeepSearchForm = () => {
...
+        mSize, email,
+        name: name.trim(),
+        dSize: dSize,
+        query: params.search
+      })
+      await fetchDeepSearch({
         mSize, email,
         name: name.trim(),
-        type: dSize,
+        dSize: dSize,
```
Comment (ivankozlovcodes) [inline src/components/deepSearchPage/submitForm/DeepSearchForm.js:90]: please user different name `dataSize`, `modelSize`

File: src/components/helpers/shuffleArray.js [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
-            var temp = array[i];
-            array[i] = array[j];
-            array[j] = temp;
-    }}
-    return array;
+function shuffleArray(array) {
+        if (array) {
+            for (var i = array.length - 1; i > 0; i--) {
+                var j = Math.floor(Math.random() * (i + 1));
+                var temp = array[i];
```
Comment (ivankozlovcodes) [inline src/components/helpers/shuffleArray.js:5]: what has changed here?

**Review by ivankozlovcodes — CHANGES_REQUESTED** [review]

File: src/components/helpers/shuffleArray.js (hunk 4 above)
Comment (dkurgan) [reply src/components/helpers/shuffleArray.js:5]: none

Comment (ivankozlovcodes) [reply src/components/helpers/shuffleArray.js:5]: Then why is this code maked as changed?

**Review by ivankozlovcodes — APPROVED** [review]

**Pull request merged**